scripts/db/src/get-user.ts

```diff
@@ -0,0 +1,74 @@
+import { Client, ConnectionConfig } from 'pg'
+
+// TODO: This is pretty copy-pasta-y from login. We should fix this by building
```

Comment:
Seems right to me, but good to think ahead. Feels like the complexity here is non-trivial.